File: workshop/codevita/prime.py



n = input()
sum=valid(n)
print sum



def valid(n):
	for i in xrange(0,n):
		if isvalid(i):
			sum+=i
			if ((sum<n)&(sum>2)):
				if(isvalid(sum)):
					count=count+1
	return count



def isvalid(num):
	if num<2:
		return 0;
	for i in xrange(2,num):
		if (num%i==0):
			return 0
	return 1

File: workshop/codevita/prime.cpp

#include<iostream.h>
int main()
{
 unsigned long int n,sum;
 cin >> n;
 sum=prime(n);
 count<<sum
 return 0;
}
 
int prime(unsigned long int n) {

 unsigned long int sum = 0,i;
 int count=0;  
for (i = 0; i <n; i++) {
     
     
   if (isPrime(i)) {
     sum += i;
     if(sum<n&&sum>2)
{
 if(isPrime(sum))
   {
     count++;
   }
}
   }
 }
 return count;
   
}



int isPrime(unsigned long int num) {
 unsigned long int i;

 
 if (num < 2) {
   return 0;
 }
   
 for (i = 2; i < num; ++i) {
   if (num % i == 0) {
     return 0;
   }
 }
 return 1;
}